src/picongpu/include/particles/filter/RelativeGlobalDomainPosition.def

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2014-2017 Rene Widera
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+namespace picongpu
+{
+namespace particles
+{
+namespace filter
+{
+namespace param
+{
+    struct RelativeGlobalDomainPosition
+    {
+        /* lowerBound is included in the range*/
+        static constexpr float_X lowerBound = 0.0;
+        /* upperBound is excluded in the range*/
+        static constexpr float_X upperBound = 1.0;
+        /* dimension for the filter
+         * x = 0; y= 1; z = 2
+         */
+        static constexpr uint32_t dimension = 0;
+    };
+} // namespace param
+
+    /** filter particle depending of the global position
+     *
+     * Check if a particle is within a relative area in one direction of the global
+     * domain.
```

Comment:
So according to our [definitions](https://github.com/ComputationalRadiationPhysics/picongpu/wiki/PIConGPU-domain-definitions), it's relative to the domain of the currently active GPUs (without window selection)?